.gitmodules

```diff
@@ -1131,6 +1131,9 @@
 [submodule "vendor/grammars/wdl-sublime-syntax-highlighter"]
 	path = vendor/grammars/wdl-sublime-syntax-highlighter
 	url = https://github.com/broadinstitute/wdl-sublime-syntax-highlighter
+[submodule "vendor/grammars/witcherscript-grammar"]
+	path = vendor/grammars/witcherscript-grammar
+	url = https://github.com/ADawesomeguy/witcherscript-grammar.git
```

Comment:
We can not accept this grammar as it's licensed under the GPL3. See pt 2 at https://github.com/github/linguist/blob/master/CONTRIBUTING.md#adding-a-language

Please find another grammar.

Reply:
Haha that was a mistake on my part since I used some stuff from and MIT licensed Visual Studio Code extension and accidentally licensed wrong. I'll just change the license :)